main.nf

```diff
@@ -0,0 +1,581 @@
+#!/usr/bin/env nextflow
+/*
+========================================================================================
+                         nf-core/deepvariant
+========================================================================================
+ nf-core/deepvariant Analysis Pipeline.
+ #### Homepage / Documentation
+ https://github.com/nf-core/deepvariant
+----------------------------------------------------------------------------------------
+*/
+
+
+def helpMessage() {
+    log.info"""
+    =========================================
+    nf-core/deepvariant v${params.pipelineVersion}
+    =========================================
+    Usage:
+
+    The typical command for running the pipeline is as follows:
+
+    nextflow run main.nf --hg19 --bam_folder "s3://deepvariant-data/test-bam/" -profile standard,docker
+
+    Mandatory arguments:
+      --bam_folder                  Path to folder containing BAM files (reads must be aligned to specified reference file, see below)
+
+    References:                     If you wish to overwrite deafult reference of hg19.
```

Comment:
For the reference, I would change that to a `--genome` instead, it would be more consistent with other nf-core pipelines, and it would make it easy to use with iGenomes